mahout/src/main/java/com/digitalpebble/behemoth/mahout/SparseVectorsFromBehemoth.java

```diff
@@ -39,14 +39,13 @@
 import org.apache.mahout.common.Pair;
 import org.apache.mahout.common.commandline.DefaultOptionCreator;
 import org.apache.mahout.math.hadoop.stats.BasicStats;
-import org.apache.mahout.vectorizer.DefaultAnalyzer;
 import org.apache.mahout.vectorizer.DictionaryVectorizer;
 import org.apache.mahout.vectorizer.DocumentProcessor;
 import org.apache.mahout.vectorizer.HighDFWordsPruner;
 import org.apache.mahout.vectorizer.collocations.llr.LLRReducer;
 import org.apache.mahout.vectorizer.common.PartialVectorMerger;
 import org.apache.mahout.vectorizer.tfidf.TFIDFConverter;
-import org.jfree.util.Log;
```

Comment:
no idea where that one came from but maybe remove the commented line altogether, no point in leaving it there